src/node/desktop/src/main/preferences/electron-desktop-options.ts

```diff
@@ -138,6 +138,11 @@ export class DesktopOptionsImpl implements DesktopOptions {
 
     if (!zoomLevel) {
       zoomLevel = this.legacyOptions.zoomLevel() ?? properties.view.default.zoomLevel;
+      const min = properties.view.properties.zoomLevel.minimum;
+      const max = properties.view.properties.zoomLevel.maximum;
+      if (zoomLevel < min || zoomLevel > max) {
```

Comment:
To confirm, this works if `zoomLevel` is `null | undefined` because it's implicitly converted to `0` here, and that's less than our configured minimum?

Reply:
Or is it just because the "legacy" zoom level is being inferred as 0 here?

Reply:
The legacy zoomLevel is returned as zero when absent. The Electron docs don't say what happens when using `systemPreferences.getUserDefault()` for a preference that doesn't exist, so I checked via debugger.

https://www.electronjs.org/docs/latest/api/system-preferences#systempreferencesgetuserdefaulttype-extends-keyof-userdefaulttypeskey-type-macos